vue/.gitignore

```diff
@@ -0,0 +1,21 @@
+.DS_Store
+node_modules
+/dist
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Editor directories and files
+.idea
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
```

Comment:
i'll probably find the answer by the time you read this, but why is there a separate .gitignore for vue?

Reply:
I didn't think about it very much. We could merge them if we want to.

Reply:
One reason to keep it separate is so we can cleanly delete this Vue directory if we decide to change frontends